install/operator/pkg/syndesis/action/pod_scheduling.go

```diff
@@ -77,7 +79,11 @@ func (a *podSchedulingAction) executeInfraScheduling(ctx context.Context, syndes
 		a.log.Error(nil, "There are no DeploymentConfig infra component to apply the infraScheduling", "label selector", selector)
 	}
 
+	// check if jaeger addon is enabled and syndesis-jaeger is installed by syndesis
+	syndesisJaegerOn := syndesis.Spec.Addons.Jaeger.Enabled && !(syndesis.Spec.Addons.Jaeger.ClientOnly && syndesis.Spec.Addons.Jaeger.OperatorOnly)
```

Comment:
Can I suggest that rather that relying on these variables, we look at the label on the pod? The syndesis-jaeger instance owned by syndesis will have  **syndesis.io/app: syndesis** whilst the OLM version does not.